frontend/uddhar-frontend/src/organization/components/AssignTeam.jsx

```
@@ -68,88 +42,109 @@ const TeamAssignment = () => {
-            isSelectable={true}
-          />
-        </div>
+    <div className="flex flex-col items-center p-8 bg-gray-200 min-h-screen w-full">
+      <Toaster position="top-center" />
+      {volunteers.length > 0 ? (
+        <>
+          <div className="flex justify-end w-full mb-4 gap-y-3">
+            <button
+              onClick={createTeam}
+              className="bg-green-500 text-white px-6 py-2 hover:bg-green-600 transition duration-200 cursor-pointer"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)